config/armv7hl/lpae

```diff
@@ -869,7 +869,7 @@ CONFIG_CRYPTO_CHACHA20_NEON=m
 CONFIG_CRYPTO_POLY1305_ARM=m
 CONFIG_CRYPTO_NHPOLY1305_NEON=m
 CONFIG_CRYPTO_CURVE25519_NEON=m
-CONFIG_AS_VFP_VMRS_FPINST=y
+# CONFIG_AS_VFP_VMRS_FPINST is not set
```

Comment:
this change is unrelated - do you have a too old gas / gcc installed?

Reply:
I did the update on my laptop, so not an armv7 gcc. That could explain it.